From aeffcdce21a314f1dd4cc6d470e6192723627720 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Wed, 1 Jun 2011 20:31:09 +0800
Subject: [PATCH 995/998] ENGR00144395-1 mx5 defconfig: disable SATA hot plug mode

This patch is used to disable SATA hot plug mode by default.
When sata device is not found, the sata internal clock (USB PHY
clock) can be disabled if hot plug mode is disabled.

Signed-off-by: Lily Zhang <r58066@freescale.com>
(cherry picked from commit 580eee65751574edc750c5da72d21cec3d70441b)
---
 arch/arm/configs/imx5_defconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index c52d711..515b845 100644
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Fri Apr 29 17:42:54 2011
+# Wed Jun  1 20:11:44 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -771,6 +771,7 @@ CONFIG_ATA_VERBOSE_ERROR=y
 # Controllers with non-SFF native interface
 #
 CONFIG_SATA_AHCI_PLATFORM=y
+CONFIG_SATA_AHCI_FSL_NO_HOTPLUG_MODE=y
 CONFIG_ATA_SFF=y
 
 #
-- 
1.7.4.1

